algorithms/integration/SConscript

```diff
@@ -35,3 +35,8 @@ env.SharedLibrary(
     source=["bayes/boost_python/ext.cc"],
     LIBS=env["LIBS"],
 )
+env.SharedLibrary(
+    target='#/lib/kapton_ext',
```

Comment:
the output target should be called `dials_algorithms_integration_kapton_ext` to match the other libraries at this level